docs/mllib-feature-extraction.md

```diff
@@ -527,6 +527,28 @@ val transformedData2 = parsedData.map(x => transformer.transform(x))
 
 {% endhighlight %}
 </div>
+
+<div data-lang="python">
+{% highlight python %}
+from pyspark import SparkContext
```

Comment:
No need to set up SparkContext for an example here.  I prefer to be able to copy and paste code into the pyspark shell for testing.